source/JsMaterialX/test/xmlIo.spec.js

```diff
@@ -244,5 +244,38 @@ describe('XmlIo', () => {
         const xmlString = mx.writeToXmlString(doc, writeOptions);
         expect(xmlString).to.include(includePath);
     });
+
+    // Node only, because we cannot read from a downloaded file in the browser
+    it('Write XML to file', async () => {
+        const filename = '_build/testFile.mtlx';
```

Comment:
This tests are exactly the same? why not run a loop or something?  Maybe we dont need XmlExport.spec?

Reply:
I actually just copied the test because I was too lazy to write a more specific one for it 😅 
Replaced it with one that actually checks the specifics of the export function 👍